SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialActorChannel.cpp

```diff
@@ -1027,8 +1026,7 @@ void USpatialActorChannel::RemoveRepNotifiesWithUnresolvedObjs(TArray<UProperty*
 {
 	// Prevent rep notify callbacks from being issued when unresolved obj references exist inside UStructs.
 	// This prevents undefined behaviour when engine rep callbacks are issued where they don't expect unresolved objects in native flow.
-	RepNotifies.RemoveAll([&](UProperty* Property)
-	{
+	RepNotifies.RemoveAll([&](UProperty* Property) {
```

Comment:
Is there a setting for this brace to be on a new line? (not a big deal if there isn't, but slight preference for brace on new line since that's more consistent with UE code)

Reply:
On a second thought, this is definitely more readable, especially inside if conditions, so let's stick with this after all :smile:

Reply:
I think you need `BreakBeforeBraces: Allman # always break before braces`. For reference see BreakBeforeBraces at https://clang.llvm.org/docs/ClangFormatStyleOptions.html

You could also test out different options quickly online - https://zed0.co.uk/clang-format-configurator/